modular_skyrat/master_files/code/modules/mob/living/simple_animal/friendly/dogs.dm

```diff
@@ -126,6 +126,11 @@
 	//spark for no reason
 	if(prob(5))
 		do_sparks(3, 1, src)
+	//dart for no reason
+	if(prob(0.01))
```

Comment:
You know this means 0.01%?

Reply:
yes, but i figured it would still be too annoying